test/integration/requests/fee.ts

```diff
@@ -0,0 +1,26 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { FeeRequest } from '../../../src'
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('Server Info Fee', function () {
+  this.timeout(TIMEOUT)
+
+  before(suiteClientSetup)
+  beforeEach(_.partial(setupClient, serverUrl))
+  afterEach(teardownClient)
+
+  it('fee', async function () {
+    const request: FeeRequest = {
+      command: 'fee',
+    }
+    const response = await this.client.request(request)
+    assert.equal(response.status, 'success')
+    assert.equal(response.type, 'response')
```

Comment:
This should also check the shape of the response object and make sure we have that right.